tests/scripts/all.sh

```diff
@@ -678,17 +678,6 @@ if uname -a | grep -F x86_64 >/dev/null; then
     scripts/config.pl unset MBEDTLS_AESNI_C
     scripts/config.pl unset MBEDTLS_PADLOCK_C
     make CC=gcc CFLAGS='-Werror -Wall -Wextra -DMBEDTLS_HAVE_INT64'
-
-    msg "test: gcc, force 64-bit compilation"
-    make test
```

Comment:
I think this paragraph should be kept.

Also, from the commit message it looks like you were meaning to add something like:
```
msg "test: gcc, force 32-bit compilation"
make test
```
above, which you didn't.

Reply:
Oh OK, this was fixed in the next commit. Cool.